storage/remote/queue_manager.go

```diff
@@ -90,13 +101,27 @@ func newQueueManagerMetrics(r prometheus.Registerer, rn, e string) *queueManager
 		Help:        "Total number of samples which failed on send to remote storage, non-recoverable errors.",
 		ConstLabels: constLabels,
 	})
+	m.failedMetadataTotal = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace:   namespace,
+		Subsystem:   subsystem,
+		Name:        "failed_metadata_total",
```

Comment:
metadata_failed would be more usual, so the metadata metrics end up together if they're sorted.

Reply:
I simply followed the naming scheme of the samples related metrics already in this file. If we want to rename even more metrics in remote write I'd prefer to do it via another PR in a later release.

Reply:
As we're already breaking the metrics here, we may aswell do all of it now rather than breaking users again in future.